tools/bin/mbedtls-check-missing-changelog

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+help () {
+  cat <<EOF
+Usage: $0 REVISION_RANGE
+Find Mbed TLS pull requests in the specified range of revisions that seem
+to be missing a changelog entry.
+
+This script uses heuristics:
+* Pull requests are merge commits on the first-parent-path in the specified
+  revision range.
+* Pull requests that modify ChangeLog or ChangeLog.d in any way are assumed
+  to be ok.
+* Pull requests whose last commit before merging has an @arm.com author
+  are assumed to be team PR where we put a changelog entry if necessary.
+EOF
+}
+
+if [[ $# -ne 1 ]]; then
+  help >&2
+  exit 120
+fi
+if [[ "$1" == '--help' ]]; then
+  help
+  exit
+fi
+
+revision_range=$1
+merge_commits=$(git log --merges --first-parent --format=%H $revision_range)
```

Comment:
Now that we always merge through GitHub, we could use a more sensitive search to only catch PR merges and not merges from inside PR.